### PR TITLE
stops megaphone sound from playing

### PR DESCRIPTION
**Pull request by hislittlecuzingames**
# About the pull request

megaphone sound is annoying. 

# Explain why it's good for the game

it's annoying. 
leaving line in so we can put in a new sound easily. 

# Testing Photographs and Procedure

<details>

</details>

# Changelog

:cl:
qol: remove megaphone sound playing
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/megaphone.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 				continue
 			listener.show_message("<B>[user]</B> broadcasts, [FONT_SIZE_LARGE("\"[message]\"")]", SHOW_MESSAGE_AUDIBLE) // 2 stands for hearable message
 			langchat_long_listeners += listener
-		playsound(loc, 'sound/items/megaphone.ogg', 100, FALSE, TRUE)
+		//playsound(loc, 'sound/items/megaphone.ogg', 100, FALSE, TRUE) // Stops playing. Commenting out so we can just plop a new sound in.
```
Comment (Zonespace27) [inline code/game/objects/items/devices/megaphone.dm:49]: if you're going to remove something, do it fully.

no comment on if the PR is good or bad

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by Blundir** [comment]
What you don't like in it exactly? I've added it. I wish we could discuss it in discord but you are not there sadly.

**Comment by hislittlecuzingames** [comment]
> What you don't like in it exactly? I've added it. I wish we could discuss it in discord but you are not there sadly.

haha yeah. 

So it's kinda "screechy." 
It kinda pierces the ears like nails on chalkboard, or a mic too close to its speaker. 
I saw a few players complain about it. 

I can see if I can apply a filter to remove some of the more "piercing" parts of the sound or something. 
But yeah those are the main complaints. 

I was working on a custom sound, but it doesn't sound "megaphone-ey" enough yet.
So just to work with your sound, I did some editing, and put a "megaphone_filtered.ogg" file in this PR that might let you hear what I took out of the sound. 

It's kinda like that defib sound that was just the defib reversed. It is a little ear piercing, and is the reason I dislike it, just someone pr-ed it before I did.

**Comment by hislittlecuzingames** [comment]
> 
> Yeah if you want to make it better, go for it, also while you are at it, i might've on accident made it only hearable to the user, it's supposed to have a big radius

I am working on some new sounds in general. 
Might do a big sound dump PR after getting feedback on sounds. 

For now, I think just removing the current sound's play code is best. 
Other people can hear the megaphone. Unsure if it was bugged in the past, but others hear it. 

@Zonespace27 ready for merge.

**Review by Blundir — CHANGES_REQUESTED** [review]
Firtly, the sound is actually only playing to the person holding it, might want to fix that, secondly, find the actual sound you've mentioned instead of removing current one

**Comment by harryob** [comment]
this pr is stale with requested changes for a couple months, ask us to reopen or make a new pr if you want to work on it

**Pull request closed without merging**